wafer/users/templates/wafer.users/snippets/profile_bio.html

```diff
@@ -0,0 +1,82 @@
+{% load i18n %}
+{% with profile=object.userprofile %}
+  <section class="wafer-profile-bio">
+    <div class="row">
+      <div class="col-md-2" id="profile-avatar">
+        {% with profile.avatar_url as avatar_url %}
+          {% if avatar_url != None %}
+            <img src="{{ profile.avatar_url }}">
+          {% endif %}
+        {% endwith %}
+        {% if can_edit %}
+          <a class="btn btn-secondary btn-sm" href="#" rel="popover" data-toggle="popover"
+              data-title="{% trans 'Changing your mugshot' %}" data-html="true"
+              data-placement="bottom">{% trans 'Edit Mugshot' %}</a>
+          <div class="popover-contents">
+            {% blocktrans %}
+              Pictures provided by <a href="https://www.libravatar.org/">libravatar</a>
+              (which falls back to <a href="https://secure.gravatar.com/">Gravatar</a>).<br>
+              Change your picture there.
+            {% endblocktrans %}
+          </div>
+        {% endif %}
+      </div>
+      <div class="col-md">
+        {% spaceless %}
+          <h1>
+            {% if profile.homepage %}
+              <a href="{{ profile.homepage_url }}">
+            {% endif %}
+            {{ profile.display_name }}
+            {% if profile.homepage %}
+              </a>
+            {% endif %}
+          </h1>
+          {% if profile.twitter_handle %}
+            <p>
+              <a href="https://twitter.com/{{ profile.twitter_handle }}" class="twitter-follow-button" data-show-count="false">
+                {% blocktrans with handle=profile.twitter_handle %}Follow @{{ handle }}{% endblocktrans %}
+              </a>
+            </p>
+          {% endif %}
+          {% if profile.github_username %}
+            <p>
+              <a href="https://github.com/{{ profile.github_username }}">
+                {% blocktrans with username=profile.github_username %}GitHub: {{ username }}{% endblocktrans %}
+              </a>
+            </p>
+          {% endif %}
+        {% endspaceless %}
+        {% if profile.bio %}
+          <div class="bio">
+            {{ profile.bio|linebreaks }}
+          </div>
+        {% endif %}
+        {% if can_edit %}
+          {% if profile.pending_talks.exists or profile.accepted_talks.exists or profile.provisional_talks.exists%}
+            {% if profile.is_registered %}
+              <div class="alert alert-success">
+                {% blocktrans %}
+                  Registered
+                {% endblocktrans %}
+              </div>
+            {% else %}
+              <div class="alert alert-danger">
+                {% blocktrans %}
+                  <strong>WARNING:</strong>
+                  Talk proposal submitted, but speaker hasn't registered to attend.
+                {% endblocktrans %}
```

Comment:
It seems a bit odd to have this bit of talk related logic here, and not in profile_talks

Perhaps we should move it so everything talk'ish is in one place.